courses/machine_learning/time_series_prediction/modeling_bqml.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>could use except(...columns to exclude...) instead. but maybe less readable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>